text-serializer-legacy/src/main/java/net/kyori/adventure/text/serializer/legacy/LegacyComponentSerializer.java

```diff
@@ -158,6 +158,15 @@ static Builder builder() {
      */
     @NonNull Builder hexColors();
 
+    /**
+     * Sets that the serializer should use the '&amp;x' repeated code format when serializing hex colors.
+     *
+     * <p>This is the format adopted by the BungeeCord (and by usage, Spigot) text API.</p>
+     *
+     * @return this builder
+     */
+    @NonNull Builder useXRepeatedCodeHexFormat();
```

Comment:
`hexColorsWithXs` ?

Reply:
Yep that works, except it's singular `x`- only one at the start of the format...